cmd/kubeadm/app/cmd/phases/init/waitcontrolplane.go

```diff
@@ -47,11 +48,18 @@ var (
 		- 'journalctl -xeu kubelet'
 
 	Additionally, a control plane component may have crashed or exited when started by the container runtime.
-	To troubleshoot, list all containers using your preferred container runtimes CLI, e.g. docker.
+	To troubleshoot, list all containers using your preferred container runtimes CLI.
+{{ if eq .RuntimeSock "crio.sock" }}
+	Here is one example how you may list all Kubernetes containers running in cri-o:
+		- 'crictl --runtime-endpoint {{ .Socket }} ps -a | grep kube | grep -v pause'
+		Once you have found the failing container, you can inspect its logs with:
+		- 'crictl --runtime-endpoint {{ .Socket }} logs CONTAINERID'
+{{ else }}
 	Here is one example how you may list all Kubernetes containers running in docker:
 		- 'docker ps -a | grep kube | grep -v pause'
 		Once you have found the failing container, you can inspect its logs with:
 		- 'docker logs CONTAINERID'
+{{ end }}
```

Comment:
printing different help per CRI, seems fine but IMO we should not use the CRISocket.